src/api/app/assets/javascripts/webui2/job_history.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+  $('#jobhistory-table').dataTable({
+    responsive: true,
+    columnDefs: [
+      { orderable: false, targets: [0, 8] },
+      { visible: false, searchable: false, targets: [1, 5] },
```

Comment:
I'm not sure I understand why the columns *1* and *5* are even needed? They are not visible, not searchable and not included in the HTML. I must be missing something. Enlighten me!

Reply:
@dmarcoux, If I'm not wrong if to order time and build time column.

Reply:
I just ported the data table structure over from the old UI. So I can not offer any enlightment.
But the [data table docu](https://datatables.net/reference/option/columnDefs.targets) says it's for making these columns not sortable.

So, right now the order of the revisions is fixed and the review links are not sortable either (which wouldn't make any sense anyway).

Reply:
We are displaying those columns sometimes? If not, I would remove them and remove this line.

Reply:
Ah, now I get what you meant. Those two columns are containing the unformatted time and build time and are used for sorting the other two columns (two and six).

Reply:
Oh, okay I understand now. It's all about the `orderData` with `targets` below. :+1: